Controller/ShortUrlController.php

```diff
@@ -15,10 +19,12 @@ class ShortUrlController extends Controller
 {
     /**
      * @param Request $request
-     * @return JsonResponse
+     * @return JsonResponse|BadRequestHttpException
```

Comment:
It throws the exception rather than returns it, you'll need to add `@throws` and put the exception there.